addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1813,6 +1813,9 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
         if (wysiwyg) {
             options.document = this.$target[0].ownerDocument;
             options.getTemplate = wysiwyg.getColorpickerTemplate.bind(wysiwyg);
+            // Load the getEditableCustomColors from the background props.
+            // This function is a common option for both background and text colorpalettes
+            options.getEditableCustomColors = wysiwyg.colorPalettesProps?.background?.getEditableCustomColors;
```

Comment:
Good, now it would be nice to add some test about this :)

1. Check if any existing test is relevant (= do we have a tour testing the colorpicker at all?)
2. If not create one, if yes consider modifying it (adding a new tour has a "big" cost for the runbot, and unfortunately, we don't have unit testing for the website yet (we'll hopefully have when we'll convert everything to OWL))
3. Try to add that kind of steps:
    - Add a background color somewhere in the page
    - Open the theme color pickers (in the theme tab)
    - Check that there is a suggestion of the background color your previously set